drivers/include/shtc1.h

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2017, RWTH Aachen. All rights reserved.
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup	drivers_shtc1
+ * @ingroup	drivers_sensors
+ * @name	Device driver interface for the SHTC1 Temperature and humidity sensor
+ * @{
+ *
+ * @file
+ * @brief	Device driver interface for the SHTC1 Temperature and humidity sensor
+ *
+ * @author	Steffen Robertz <steffen.robertz@rwth-aachen.de>
+ * @author	Josua Arndt <steffen.robertz@rwth-aachen.de>
+ */
+
+#ifndef SHTC1_H
+#define SHTC1_H
+
+#include <stdint.h>
+#include "periph/i2c.h"
+
+#ifdef __cplusplus 
+extern "C" {
+#endif
+
+typedef enum {
+    CRC_DISABLED = 0,
+    CRC_ENABLED
+}crc_type_t;
+
+typedef struct {
+    float temp;
+    float rel_humidity;
+}shtc1_values_t;
+
+
+/**
+ * @brief initializes the sensor and i2c
+ *
+ * @param[in] dev		i2c device to be used
+ *
+ * @return			0 on a working initialization
+ * @return			-1 on undefined i2c device given in periph_conf
+ * @return			-2 on unsupported speed value
+ */
+int8_t shtc1_init(i2c_t dev);
```

Comment:
Please read the wiki page [Writing-a-device-driver-in-RIOT](https://github.com/RIOT-OS/RIOT/wiki/Guide:-Writing-a-device-driver-in-RIOT) and adapt accordingly. Grep for "device descriptor" and "device configuration" (params)